department-of-veterans-affairs/caseflow-feedback
# Add PII column to admin view and fix css

Connects to #165 

To test it: 

- Submit feedback including PII information 
- Go to /admin
- Make sure PII column shows submitted PII information 

<img width="1018" alt="screen shot 2018-04-09 at 1 48 53 pm" src="https://user-images.githubusercontent.com/4960030/38513700-05a67f2a-3bfd-11e8-8da2-b1718c8bbc42.png">
